### DISCUSSION TITLE
How to get the active pane's ID?

### DISCUSSION THREAD

**Discussion by weiying-chen** [wezterm/wezterm#5010] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Wayland/Mutter

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I thought $WEZTERM_PANE would get the activate pane (current pane's ID). But it gets the ID of the pane that is running the script (e.g. inside inotifywait).

How to get the active (focused) pane's ID?

**Comment by wez** [comment, marked as answer]
`wezterm cli list-clients` (or `wezterm cli list-clients --format=json`) will show you the focused pane for each client.

**Reply by weiying-chen** [reply]
I could find the `is_active` field with this: `wezterm cli list --format=json`. Thanks!

**Reply by dw72** [reply]
What when we have run ssh to other machine in that pane? How to identify it when there is no wezterm installed on the remote?
